### ISSUE TITLE
[Issue]: Apply system settings saves sampler parameters

### ISSUE THREAD

**Issue by IowaSovereign** [vladmandic/automatic#3641]
### Issue Description

Certain parameters present in the generation UI are saved into config.txt when "Apply settings" button is used in the system settings. Any parameters which are on the left-hand side UI (using modern UI terminology) should not be saved by the "Apply settings" just like prompts, chosen sampler, etc. aren't, and should apply dynamically as user changes them, and if user wants to save them "UI defaults" should be used. As an example, as it is now detailer params get saved into the config, but second pass parameters like upscale aren't, which is confusing.

There is also a second issue related to this, and at least the advanced sampler parameters, such as manually setting prediction type, turning on rescale and dynamic threshold get saved into the config.txt but you can't turn those off or set them back to default,
`schedulers_prediction_type"v_prediction"
 schedulers_use_thresholding true
 schedulers_rescale_betas true` will remain in the config even if you tick them off, requiring you to manually delete the lines from the config, not even saving the settings again after changing them back will turn them off. Again, since these are present in the generation UI they should not be affected by "Apply settings".

### Version Platform Description

Windows 11 24H2, GTX 1080Ti 11Gb
Microsoft Edge browser

Issue not related to platform

### Backend

Diffusers

### UI

ModernUI

### Branch

Master

### Model

StableDiffusion XL

### Acknowledgements

- [X] I have read the above and searched for existing issues
- [X] I confirm that this is classified correctly and its not an extension issue

**Comment by vladmandic** [comment]
> Any parameters which are on the left-hand side UI (using modern UI terminology) should not be saved by the "Apply settings" just like prompts, chosen sampler, etc. aren't, and should apply dynamically as user changes them, and if user wants to save them "UI defaults" should be used

that is true in theory. in practice, scheduler options are exception and its documented in changelog.
![image](https://github.com/user-attachments/assets/14e70752-ab3c-4482-b489-4c08c5f2b759)

for this part, if you really want to change this, i suggest to create a feature request as this works-as-documented.

> turning on rescale and dynamic threshold get saved into the config.txt but you can't turn those off or set them back to default,
schedulers_prediction_type"v_prediction" schedulers_use_thresholding true schedulers_rescale_betas true will remain in the config even if you tick them off, requiring you to manually delete the lines from the config

this would be an issue, but i cannot reproduce it. 
for example, as soon as i change prediction type back to default, the line from config.json gets removed.

please provide log or some kind of proof and/or reproduction showing this?

**Comment by IowaSovereign** [comment]
Well, them being an exception (and not the only one, as I said detailer also does this) is the point I made, for user friendliness it would be better to move them back where they were before or make them behave like the rest of that UI so they don't create confusion. Also changelog wouldn't be a place for any user to look for information about UI would it? A new user definitely wouldn't. It's anecdotal but I introduced some friends to SD.Next and telling them "yeah, this is different than all the other things in this section of the UI, you just gotta remember it" doesn't feel right. Is it a lot of work, impossible or just a conscious design choice you made? I'm just looking at it from my friends' perspective.

As for the second issue, this time it actually broke a different way than the one I described when I tried to reproduce it, I attached a sped-up video to show you the exact behaviour. This time the settings didn't apply at all and didn't save; it also didn't affect the image.

https://github.com/user-attachments/assets/1e320061-5397-43a0-b897-6dfd3e91dd80

**Comment by IowaSovereign** [comment]
Actually, since we're on this topic, same thing happens to my detailer settings, after saving settings my detailer_confidence stays at 0, and is not saved or the changes used by the detailer, it stays at 0.

**Comment by vladmandic** [comment]
> I will make a feature request just so it doesn't fall of the radar.

i'm not disagreeing, but unless someone steps up, its going to remain open.

on why? it was a decision to avoid huge amount of work and likely break every extension that relies on accessing main processing functions.
reason is that ui variables are passed as params and have to propagate throughout the stack - which means adding 6 new variables in 12 function signatures (12 for each of text/image/control).
and settings can be accessed as-is. so i choose to use settings.
or alternative is not to have those settings in main ui and move them deep into settings menus - does that help anyone?

> Update, even worse, after editing it manually to 0.4 and then setting a new value in the UI it actually sets it back to 0 after "Apply settings"

i cannot reproduce any of those issues.

**Comment by vladmandic** [comment]
fixed. changing settings worked as expected (and saved immediately), but when doing manual "apply settings", it was reset.
